roachtest: sqlsmith/setup=tpch-sf1/setting=default failed

[(roachtest).sqlsmith/setup=tpch-sf1/setting=default failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1690462&tab=buildLog) on [release-19.2@99a4ab5a34b310338098a6aafb6621920fd9d4cc](https://github.com/cockroachdb/cockroach/commits/99a4ab5a34b310338098a6aafb6621920fd9d4cc):

```
The test failed on branch=release-19.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200115-1690462/sqlsmith/setup=tpch-sf1/setting=default/run_1
	sqlsmith.go:188,sqlsmith.go:193,sqlsmith.go:226,test_runner.go:716: query timed out, but did not cancel execution:
		SELECT
			tab_38.p_comment AS col_70, tab_38.p_retailprice AS col_71, tab_39.s_address AS col_72
		FROM
			defaultdb.public.part@[0] AS tab_38 CROSS JOIN defaultdb.public.supplier@primary AS tab_39;
```

<details><summary>Repro</summary><p>


Artifacts: [/sqlsmith/setup=tpch-sf1/setting=default](https://teamcity.cockroachdb.com/viewLog.html?buildId=1690462&tab=artifacts#/sqlsmith/setup=tpch-sf1/setting=default)
```
make stressrace TESTS=sqlsmith/setup=tpch-sf1/setting=default PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).sqlsmith/setup=tpch-sf1/setting=default failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2041122&tab=buildLog) on [master@d3791a81c0716478de08d44459d3fcf5b4f3ea1e](https://github.com/cockroachdb/cockroach/commits/d3791a81c0716478de08d44459d3fcf5b4f3ea1e):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/sqlsmith/setup=tpch-sf1/setting=default/run_1
	sqlsmith.go:169,sqlsmith.go:199,test_runner.go:753: error: pq: internal error: column 3 not in input
		stmt:
		SELECT
			tab_1.ps_partkey AS col_1,
			tab_1.ps_partkey AS col_2,
			tab_1.ps_supplycost AS col_3,
			NULL AS col_4,
			tab_1.ps_suppkey AS col_5
		FROM
			defaultdb.public.partsupp@ps_sk AS tab_1
		ORDER BY
			tab_1.ps_availqty DESC
		LIMIT
			78:::INT8;

```

<details><summary>More</summary><p>


Artifacts: [/sqlsmith/setup=tpch-sf1/setting=default](https://teamcity.cockroachdb.com/viewLog.html?buildId=2041122&tab=artifacts#/sqlsmith/setup=tpch-sf1/setting=default)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Asqlsmith%2Fsetup%3Dtpch-sf1%2Fsetting%3Ddefault.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).sqlsmith/setup=tpch-sf1/setting=default failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2231078&tab=buildLog) on [master@8b91062f9351d18f9104aff567cb152df162021e](https://github.com/cockroachdb/cockroach/commits/8b91062f9351d18f9104aff567cb152df162021e):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/sqlsmith/setup=tpch-sf1/setting=default/run_1
	sqlsmith.go:172,sqlsmith.go:202,test_runner.go:754: error: pq: internal error: runtime error: invalid memory address or nil pointer dereference
		stmt:
		SELECT
			tab_32.l_quantity AS col_66,
			e'\x19z}\x15>R ':::STRING AS col_67,
			tab_32.l_linestatus AS col_68,
			NULL AS col_69,
			(-7346679947572328759):::INT8 AS col_70,
			tab_32.l_suppkey AS col_71,
			0.6062107066527916:::FLOAT8 AS col_72
		FROM
			defaultdb.public.lineitem@l_sd AS tab_32
			JOIN defaultdb.public.supplier AS tab_33
				JOIN defaultdb.public.orders AS tab_34 ON (tab_33.s_suppkey) = (tab_34.o_orderkey) ON
					(tab_32.l_orderkey) = (tab_33.s_suppkey) AND (tab_32.l_linenumber) = (tab_33.s_suppkey)
		WHERE
			true
		ORDER BY
			tab_32.tableoid ASC, tab_32.l_shipmode, tab_32.l_shipdate ASC, tab_33.s_suppkey ASC;

```

<details><summary>More</summary><p>


Artifacts: [/sqlsmith/setup=tpch-sf1/setting=default](https://teamcity.cockroachdb.com/viewLog.html?buildId=2231078&tab=artifacts#/sqlsmith/setup=tpch-sf1/setting=default)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Asqlsmith%2Fsetup%3Dtpch-sf1%2Fsetting%3Ddefault.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

--- a/pkg/cmd/roachtest/backup.go
+++ b/pkg/cmd/roachtest/backup.go
@@ -195,7 +195,7 @@ func registerBackup(r *testRegistry) {
 				// the average MB/sec per node.
 				tick()
 				c.Run(ctx, c.Node(1), `./cockroach sql --insecure -e "
-				BACKUP bank.bank TO 'gs://cockroachdb-backup-testing/`+dest+`'"`)
+				BACKUP bank.bank TO 'gs://cockroachdb-backup-testing/`+dest+`?AUTH=implicit'"`)
 				tick()
 
 				// Upload the perf artifacts to any one of the nodes so that the test
@@ -348,7 +348,7 @@ func registerBackup(r *testRegistry) {
 			}
 			warehouses := 10
 
-			backupDir := "gs://cockroachdb-backup-testing/" + c.name
+			backupDir := "gs://cockroachdb-backup-testing/" + c.name + "?AUTH=implicit"
 			// Use inter-node file sharing on 20.1+.
 			if r.buildVersion.AtLeast(version.MustParse(`v20.1.0-0`)) {
 				backupDir = "nodelocal://1/" + c.name

--- a/pkg/cmd/roachtest/cdc.go
+++ b/pkg/cmd/roachtest/cdc.go
@@ -87,7 +87,7 @@ func cdcBasicTest(ctx context.Context, t *test, c *cluster, args cdcTestArgs) {
 		ts := timeutil.Now().Format(`20060102150405`)
 		// cockroach-tmp is a multi-region bucket with a TTL to clean up old
 		// data.
-		sinkURI = `experimental-gs://cockroach-tmp/roachtest/` + ts
+		sinkURI = `experimental-gs://cockroach-tmp/roachtest/` + ts + "?AUTH=implicit"
 	} else {
 		t.Status("installing kafka")
 		kafka.install(ctx)

--- a/pkg/cmd/roachtest/import.go
+++ b/pkg/cmd/roachtest/import.go
@@ -32,16 +32,16 @@ func registerImportNodeShutdown(r *testRegistry) {
 			}
 			importStmt := fmt.Sprintf(`
 				IMPORT TABLE %[1]s
-				CREATE USING 'gs://cockroach-fixtures/tpch-csv/schema/%[1]s.sql'
+				CREATE USING 'gs://cockroach-fixtures/tpch-csv/schema/%[1]s.sql?AUTH=implicit'
 				CSV DATA (
-				'gs://cockroach-fixtures/tpch-csv/sf-100/%[1]s.tbl.1',
-				'gs://cockroach-fixtures/tpch-csv/sf-100/%[1]s.tbl.2',
-				'gs://cockroach-fixtures/tpch-csv/sf-100/%[1]s.tbl.3',
-				'gs://cockroach-fixtures/tpch-csv/sf-100/%[1]s.tbl.4',
-				'gs://cockroach-fixtures/tpch-csv/sf-100/%[1]s.tbl.5',
-				'gs://cockroach-fixtures/tpch-csv/sf-100/%[1]s.tbl.6',
-				'gs://cockroach-fixtures/tpch-csv/sf-100/%[1]s.tbl.7',
-				'gs://cockroach-fixtures/tpch-csv/sf-100/%[1]s.tbl.8'
+				'gs://cockroach-fixtures/tpch-csv/sf-100/%[1]s.tbl.1?AUTH=implicit',
+				'gs://cockroach-fixtures/tpch-csv/sf-100/%[1]s.tbl.2?AUTH=implicit',
+				'gs://cockroach-fixtures/tpch-csv/sf-100/%[1]s.tbl.3?AUTH=implicit',
+				'gs://cockroach-fixtures/tpch-csv/sf-100/%[1]s.tbl.4?AUTH=implicit',
+				'gs://cockroach-fixtures/tpch-csv/sf-100/%[1]s.tbl.5?AUTH=implicit',
+				'gs://cockroach-fixtures/tpch-csv/sf-100/%[1]s.tbl.6?AUTH=implicit',
+				'gs://cockroach-fixtures/tpch-csv/sf-100/%[1]s.tbl.7?AUTH=implicit',
+				'gs://cockroach-fixtures/tpch-csv/sf-100/%[1]s.tbl.8?AUTH=implicit'
 				) WITH  delimiter='|', detached
 			`, tableName)
 			gatewayDB := c.Conn(ctx, gatewayNode)
@@ -233,16 +233,16 @@ func registerImportTPCH(r *testRegistry) {
 					defer t.WorkerStatus()
 					_, err := conn.Exec(`
 				IMPORT TABLE csv.lineitem
-				CREATE USING 'gs://cockroach-fixtures/tpch-csv/schema/lineitem.sql'
+				CREATE USING 'gs://cockroach-fixtures/tpch-csv/schema/lineitem.sql?AUTH=implicit'
 				CSV DATA (
-				'gs://cockroach-fixtures/tpch-csv/sf-100/lineitem.tbl.1',
-				'gs://cockroach-fixtures/tpch-csv/sf-100/lineitem.tbl.2',
-				'gs://cockroach-fixtures/tpch-csv/sf-100/lineitem.tbl.3',
-				'gs://cockroach-fixtures/tpch-csv/sf-100/lineitem.tbl.4',
-				'gs://cockroach-fixtures/tpch-csv/sf-100/lineitem.tbl.5',
-				'gs://cockroach-fixtures/tpch-csv/sf-100/lineitem.tbl.6',
-				'gs://cockroach-fixtures/tpch-csv/sf-100/lineitem.tbl.7',
-				'gs://cockroach-fixtures/tpch-csv/sf-100/lineitem.tbl.8'
+				'gs://cockroach-fixtures/tpch-csv/sf-100/lineitem.tbl.1?AUTH=implicit',
+				'gs://cockroach-fixtures/tpch-csv/sf-100/lineitem.tbl.2?AUTH=implicit',
+				'gs://cockroach-fixtures/tpch-csv/sf-100/lineitem.tbl.3?AUTH=implicit',
+				'gs://cockroach-fixtures/tpch-csv/sf-100/lineitem.tbl.4?AUTH=implicit',
+				'gs://cockroach-fixtures/tpch-csv/sf-100/lineitem.tbl.5?AUTH=implicit',
+				'gs://cockroach-fixtures/tpch-csv/sf-100/lineitem.tbl.6?AUTH=implicit',
+				'gs://cockroach-fixtures/tpch-csv/sf-100/lineitem.tbl.7?AUTH=implicit',
+				'gs://cockroach-fixtures/tpch-csv/sf-100/lineitem.tbl.8?AUTH=implicit'
 				) WITH  delimiter='|'
 			`)
 					return errors.Wrap(err, "import failed")

--- a/pkg/cmd/roachtest/restore.go
+++ b/pkg/cmd/roachtest/restore.go
@@ -228,7 +228,7 @@ func registerRestoreNodeShutdown(r *testRegistry) {
 
 				// 10 GiB restore.
 				restoreQuery := `RESTORE bank.bank FROM
-					'gs://cockroach-fixtures/workload/bank/version=1.0.0,payload-bytes=100,ranges=10,rows=10000000,seed=1/bank'`
+					'gs://cockroach-fixtures/workload/bank/version=1.0.0,payload-bytes=100,ranges=10,rows=10000000,seed=1/bank?AUTH=implicit'`
 
 				t.l.Printf("starting to run the restore job")
 				if _, err := gatewayDB.Exec(restoreQuery); err != nil {
@@ -373,7 +373,7 @@ func registerRestore(r *testRegistry) {
 					tick()
 					c.Run(ctx, c.Node(1), `./cockroach sql --insecure -e "
 				RESTORE csv.bank FROM
-				'gs://cockroach-fixtures/workload/bank/version=1.0.0,payload-bytes=10240,ranges=0,rows=65104166,seed=1/bank'
+				'gs://cockroach-fixtures/workload/bank/version=1.0.0,payload-bytes=10240,ranges=0,rows=65104166,seed=1/bank?AUTH=implicit'
 				WITH into_db = 'restore2tb'"`)
 					tick()
 

--- a/pkg/cmd/roachtest/schemachange.go
+++ b/pkg/cmd/roachtest/schemachange.go
@@ -26,7 +26,7 @@ func registerSchemaChangeDuringKV(r *testRegistry) {
 		Owner:   OwnerSQLSchema,
 		Cluster: makeClusterSpec(5),
 		Run: func(ctx context.Context, t *test, c *cluster) {
-			const fixturePath = `gs://cockroach-fixtures/workload/tpch/scalefactor=10/backup`
+			const fixturePath = `gs://cockroach-fixtures/workload/tpch/scalefactor=10/backup?AUTH=implicit`
 
 			c.Put(ctx, cockroach, "./cockroach")
 			c.Put(ctx, workload, "./workload")

--- a/pkg/cmd/roachtest/sqlsmith.go
+++ b/pkg/cmd/roachtest/sqlsmith.go
@@ -30,7 +30,7 @@ func registerSQLSmith(r *testRegistry) {
 		"seed":        sqlsmith.Setups["seed"],
 		"rand-tables": sqlsmith.Setups["rand-tables"],
 		"tpch-sf1": func(r *rand.Rand) string {
-			return `RESTORE TABLE tpch.* FROM 'gs://cockroach-fixtures/workload/tpch/scalefactor=1/backup' WITH into_db = 'defaultdb';`
+			return `RESTORE TABLE tpch.* FROM 'gs://cockroach-fixtures/workload/tpch/scalefactor=1/backup?AUTH=implicit' WITH into_db = 'defaultdb';`
 		},
 		"tpcc": func(r *rand.Rand) string {
 			const version = "version=2.1.0,fks=true,interleaved=false,seed=1,warehouses=1"
@@ -46,7 +46,7 @@ func registerSQLSmith(r *testRegistry) {
 				"stock",
 				"warehouse",
 			} {
-				fmt.Fprintf(&sb, "RESTORE TABLE tpcc.%s FROM 'gs://cockroach-fixtures/workload/tpcc/%[2]s/%[1]s' WITH into_db = 'defaultdb';\n", t, version)
+				fmt.Fprintf(&sb, "RESTORE TABLE tpcc.%s FROM 'gs://cockroach-fixtures/workload/tpcc/%[2]s/%[1]s?AUTH=implicit' WITH into_db = 'defaultdb';\n", t, version)
 			}
 			return sb.String()
 		},

--- a/pkg/cmd/roachtest/tpc_utils.go
+++ b/pkg/cmd/roachtest/tpc_utils.go
@@ -67,7 +67,7 @@ func loadTPCHDataset(
 	}
 
 	t.l.Printf("restoring tpch scale factor %d\n", sf)
-	tpchURL := fmt.Sprintf("gs://cockroach-fixtures/workload/tpch/scalefactor=%d/backup", sf)
+	tpchURL := fmt.Sprintf("gs://cockroach-fixtures/workload/tpch/scalefactor=%d/backup?AUTH=implicit", sf)
 	query := fmt.Sprintf(`CREATE DATABASE IF NOT EXISTS tpch; RESTORE tpch.* FROM '%s' WITH into_db = 'tpch';`, tpchURL)
 	_, err := db.ExecContext(ctx, query)
 	return err

--- a/pkg/cmd/roachtest/tpcdsvec.go
+++ b/pkg/cmd/roachtest/tpcdsvec.go
@@ -59,7 +59,7 @@ func registerTPCDSVec(r *testRegistry) {
 		disableAutoStats(t, clusterConn)
 		t.Status("restoring TPCDS dataset for Scale Factor 1")
 		if _, err := clusterConn.Exec(
-			`RESTORE DATABASE tpcds FROM 'gs://cockroach-fixtures/workload/tpcds/scalefactor=1/backup';`,
+			`RESTORE DATABASE tpcds FROM 'gs://cockroach-fixtures/workload/tpcds/scalefactor=1/backup?AUTH=implicit';`,
 		); err != nil {
 			t.Fatal(err)
 		}

